start.sh

```diff
@@ -58,23 +58,17 @@ echo 'WP_SITEURL=${WP_HOME}/wp' >> .env;
 read -p "Enter ACF PRO KEY: " acfprokey
 echo "ACF_PRO_KEY=$acfprokey" >> .env
 echo "#/ ENV" >> .env;
-echo "" >> .env;
-echo "" >> .env;
+echo >> .env;
+echo >> .env;
 
 htaccess="$(cat '.htaccess')"
 > '.htaccess'
 echo "$htaccess" | sed -r "s/juniper.local/$domain/g" >> '.htaccess'
 
 echo "# SALTS" >> .env;
-wget -qO /tmp/wp.keys https://api.wordpress.org/secret-key/1.1/salt/
-echo "AUTH_KEY=\"$(cat /tmp/wp.keys |grep -w AUTH_KEY | cut -d \' -f 4)\"" >>  .env
-echo "SECURE_AUTH_KEY=\"$(cat /tmp/wp.keys |grep -w SECURE_AUTH_KEY | cut -d \' -f 4)\"" >>  .env
-echo "LOGGED_IN_KEY=\"$(cat /tmp/wp.keys |grep -w LOGGED_IN_KEY | cut -d \' -f 4)\"" >>  .env
-echo "NONCE_KEY=\"$(cat /tmp/wp.keys |grep -w NONCE_KEY | cut -d \' -f 4)\"" >>  .env
-echo "AUTH_SALT=\"$(cat /tmp/wp.keys |grep -w AUTH_SALT | cut -d \' -f 4)\"" >>  .env
-echo "SECURE_AUTH_SALT=\"$(cat /tmp/wp.keys |grep -w SECURE_AUTH_SALT | cut -d \' -f 4)\"" >>  .env
-echo "LOGGED_IN_SALT=\"$(cat /tmp/wp.keys |grep -w LOGGED_IN_SALT | cut -d \' -f 4)\"" >>  .env
-echo "NONCE_SALT=\"$(cat /tmp/wp.keys |grep -w NONCE_SALT | cut -d \' -f 4)\"" >>  .env
+wget -qO - https://api.wordpress.org/secret-key/1.1/salt/ \
+  | tr "'" '"' \
+  | sed -n -e 's#^define("\([A-Z_]\+\)",\s\+\("[^"]\+"\));$#\1=\2#p' >> .env
```

Comment:
`tr` changes single quotes to double quotes.
`sed` removes `define( ... );`.

Reply:
Have you tried to fire it on your localhost? It looks like it's not working for me :(
What OS are you using?

Reply:
I'm lock in the golden cage of Debian Linux.
Do you have non-GNU sed on macOS?

Reply:
Yes, and as far as I can see we just found a compatibility issue here :| Sed which is used for .htaccess in the start.sh does not work for me either.

I guess that we can solve this with https://stackoverflow.com/a/41029130 but I think that doing it in the way it is right now gets rid of this problem :|

Reply:
There can be dragons outside POSIX!

Reply:
I'm going to try `--posix` which disables non-POSIX features.

Reply:
... is fails miserably!

Reply:
Does your .env processor accept `$` in the quoted value?

Reply:
`\s` was the only non-POSIX feature. Now your macOS sed should run!

Reply:
> There can be dragons outside POSIX!

..and beautiful princesses!  Thanks for your fix :) I am going to merge it soon because it is working of course.